miio/walkingpad.py

```diff
@@ -0,0 +1,189 @@
+import enum
+import logging
+from collections import defaultdict
+from typing import Any, Dict
+
+import click
+
+from .click_common import EnumType, command, format_output
+from .device import Device, DeviceStatus
+from .exceptions import DeviceException
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class WalkingpadException(DeviceException):
+    pass
+
+
+class OperationMode(enum.Enum):
+    Auto = 0
+    Manual = 1
+    Off = 2
+
+
+class WalkingpadStatus(DeviceStatus):
+    """Container for status reports from Xiaomi Walkingpad."""
```

Comment:
Please add an example of the input data dictionary which is used to initialize this class. The simplest way is likely to add a temporary `from pprint import pprint; pprint(self.data)` line inside `__init__` and copy&paste it to the docstring.